<?xml version="1.0" ?>
<launch>
  <arg name="robot" default="panda" doc="choose your robot. Possible values: [panda, fr3]"/>
  <arg name="arm_id" default="$(arg robot)" />
  <include file="$(find franka_control)/launch/franka_control.launch" pass_all_args="true"/>
  <rosparam command="load" file="$(find pandaController)/config/pandaController.yaml" subst_value="true" />
  <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen" args="PandaController"/>
  <node pkg="rviz" type="rviz" output="screen" name="rviz" args="-d $(find pandaController)/launch/rviz/franka_description_with_marker.rviz"/>
  <!-- <node name="interactive_marker" pkg="pandaController" type="interactive_marker.py" required="true" output="screen">
    <param name="link_name" value="$(arg arm_id)_link0" />
  </node> -->
  <node name="rqt_reconfigure" pkg="rqt_reconfigure" type="rqt_reconfigure" required="false" />
</launch>
